### PR TITLE
Issue 38 replace deprecated bower

### PR DESCRIPTION
**Pull request by cristiantrave**
First steps for webpack implementation in rosetta. Removed bower. Now the libraries are managed by npm.

### CODE REVIEW

File: rosetta/package.json [hunk 1]
```diff
@@ -2,10 +2,18 @@
   "name": "rosetta",
   "version": "0.1.0",
   "devDependencies": {
+    "backbone": "^1.3.3",
+    "underscore": "^1.8.3",
+    "jquery": "^2.2.3",
+    "handlebars": "^4.0.5",
+    "bootstrap": "^3.3.6",
+    "normalize-css": "^2.3.1",
```
Comment (lluiscampos) [inline rosetta/package.json]: Why do we downgrade normalize?

File: rosetta/bower.json [hunk 2]
```diff
@@ -1,30 +0,0 @@
-{
```
Comment (lluiscampos) [inline rosetta/bower.json]: We also need to remove bower from the documentation (README.md file)

File: rosetta/Gruntfile.js [hunk 3]
```diff
@@ -107,13 +94,12 @@ module.exports = function(grunt) {
...
   grunt.loadNpmTasks('grunt-concurrent');
-  grunt.loadNpmTasks('grunt-bowercopy');
   grunt.loadNpmTasks('grunt-json-server');
   grunt.loadNpmTasks('grunt-browserify');
   grunt.loadNpmTasks('grunt-mocha-phantom-istanbul');
 
   grunt.registerTask('lint', ['eslint']);
-  grunt.registerTask('build', ['copy:app', 'bowercopy']);
+  grunt.registerTask('build', ['copy:app']);
   grunt.registerTask('test', ['browserify:coverage', 'mocha']);
```
Comment (lluiscampos) [inline rosetta/Gruntfile.js]: The reason for the test to fail is that they refer to non-existing files in bower_components. We need to modify file rosetta/test/load.js

**Review by lluiscampos — CHANGES_REQUESTED** [review]
See my comments

File: rosetta/package.json (hunk 1 above)
Comment (cristiantrave) [inline rosetta/package.json]: Npm says that it is the highest version of which it has references. The message comes up doing npm install command. Performing  tests are normal with this lower version.

File: rosetta/bower.json (hunk 2 above)
Comment (cristiantrave) [inline rosetta/bower.json]: Now bower references in the README have been deleted

File: rosetta/Gruntfile.js (hunk 3 above)
Comment (cristiantrave) [inline rosetta/Gruntfile.js]: True! Corrected.

**Review by lluiscampos — APPROVED** [review]
Great work :)

**Pull request merged**